roles/addons/prometheus_server/tasks/main.yml

```diff
@@ -44,6 +44,20 @@
   tags:
     - firewall
 
+- name: "copy █ Install karma firewalld service configuration file"
```

Comment:
I would copy both files alertmanager.xml and karma.xml in the same task, with a loop.

Off the top of my head:
```
copy:
  src: {{ item }}
  dest:  "/etc/firewalld/services/{{ item }}"
[...]
loop: ['alertmanager.xml', 'karma.xml']
```